src/popup/pages/Popups/ConnectBase.vue

```diff
@@ -0,0 +1,318 @@
+<template>
+  <Modal
+    show
+    full-screen
+    class="connect"
+    data-cy="popup-aex2"
+    semi-dense
+  >
+    <p
+      class="text-heading-4 text-center header"
+      data-cy="page-header"
+      v-text="$t('pages.connectConfirm.title')"
+    />
+    <p
+      class="text-center"
+      v-text="$t('pages.connectConfirm.subTitle')"
+    />
+
+    <div class="connect-parties">
+      <p
+        v-if="isUnknown"
+        class="warning-message color-warning"
+        v-text="$t('pages.connectConfirm.unknownOriginWarning')"
+      />
+
+      <Card
+        v-if="sender"
+        data-cy="aepp"
+        :text="sender.name || $t('pages.connectConfirm.unknownSource')"
+        :description="sender.url || $t('pages.connectConfirm.unknownUrl')"
+        :variant="(isUnknown) ? 'warning' : undefined"
+        icon-centered
+        dense
+      >
+        <template #icon>
+          <Avatar size="rg" borderless>
+            <img v-if="appIcon" :src="appIcon" class="dapp-logo" alt="DAPP logo" />
+            <DappIcon v-else class="dapp-icon" />
+          </Avatar>
+        </template>
+      </Card>
+
+      <div class="connect-parties-label">
+        <div class="line">
+          <TriangleRightIcon class="triangle" />
+        </div>
+        <p
+          class="text-description text-center"
+          v-text="$t('pages.connectConfirm.websiteRequestConnect')"
+        />
+      </div>
+
+      <AccountSelectOptionsItem
+        v-if="activeAccount"
+        :custom-account="activeAccount"
+        hide-balance
+        hide-protocol-icon
+      />
+      <Card v-else dense>
+        <AnimatedSpinnerIcon class="spinner" />
+      </Card>
+    </div>
+
+    <p
+      class="text-description text-center permissions-header"
+      v-text="$t('pages.connectConfirm.permissionsHeader')"
+    />
+
+    <div class="permissions">
+      <div v-for="(accessName, index) in access" :key="index">
+        <div class="label">
+          <CheckMark class="icon" />
+          {{ accessLabels[accessName]?.label || 'Unknown' }}
+        </div>
+        <div class="description" v-text="accessLabels[accessName]?.description" />
```

Comment:
Do we want to continue splitting every property on a new line in the template or shall we from now on no longer follow this pattern?

Reply:
The ESLint settings allows us to put more than one property in single line. I think we can use this possibility, but in this case you're right. I'll split this line to match the common pattern in this file.